### ISSUE TITLE
Availability of test vibration mode shapes

### ISSUE THREAD

**Issue by nilswagner** [PAAW/mAEWing2#1]
I am just curious. Are the experimental mode shapes from the GVT part of the git repository?
If not, is it possible to make them available in Universal File Format?

**Comment by pjseiler** [comment]
Let me check to see if they are posted somewhere on the GitHub.  If
not then we have Matlab scripts that can
be used to regenerate them.  Matlab created an example based on this
GVT data that you might find useful:
   https://www.mathworks.com/help/ident/examples/modal-analysis-of-a-flexible-flying-wing-aircraft.html
This contains the code to generate and animate these mode shapes.

**Comment by nilswagner** [comment]
Are you aware of http://hosting.umons.ac.be/html/mecara/EasyMod/index.html ?

unv55write(infoMODE,filename,ind_complex): This function writes the information
about the modal parameters in a 55 type UFF file.

unv15and82write(Nodes,Connexions,filename,NodeColor,LineColor): This func-
tion records the geometry information (nodes and connexions) in UFF file bringing
together the file type 15 (for the nodes) and the file type 82 (for the connexions between
nodes).

Your support is greatly appreciated. Thanks in advance.

**Comment by abhineet-gupta** [comment]
The scripts to obtain experimental mode shapes have been added [here](https://github.com/PAAW/mAEWing2/tree/master/GVT/Huggin).

Let us know if this resolves the issue.

**Comment by nilswagner** [comment]
Yeah, but I don't have a Matlab license right now. Hence the question about Universal files.

http://sdrl.uc.edu/sdrl/referenceinfo/universalfileformats/file-format-storehouse/universal-dataset-number-15
http://sdrl.uc.edu/sdrl/referenceinfo/universalfileformats/file-format-storehouse/universal-dataset-number-55
http://sdrl.uc.edu/sdrl/referenceinfo/universalfileformats/file-format-storehouse/universal-dataset-number-82

**Comment by abhineet-gupta** [comment]
I am not used to this file format but I have saved the resulting mode shapes in the file below.
[ModeShapes.txt](https://github.com/PAAW/mAEWing2/files/3794481/ModeShapes.txt)

The columns in the file are the mode shapes. The corresponding frequencies and locations can be read from the matlab files which can be opened in any text editor.

**Comment by nilswagner** [comment]
I prefer Python

import numpy as np
X = np.loadtxt('ModeShapes.txt', delimiter=',')
print (X.shape)
>>> X.shape
(30, 8)

You have extracted 8 mode shapes. The number of sensors (tri-axial accelerometers) is ten, right?

**Comment by abhineet-gupta** [comment]
There are 30 single axis accelerometers. I have extracted 8 mode shapes.

**Comment by nilswagner** [comment]
![mode_01](https://user-images.githubusercontent.com/5382885/68129278-90cb0400-ff19-11e9-8e2c-ff8a4ec24de8.gif)

[wing.zip](https://github.com/PAAW/mAEWing2/files/3804489/wing.zip)
The z-coordinates of the 30 single-axis accelerometers should be different due to the curved wing, right?

**Comment by gupta165** [comment]
We ignored the affect of curvature of the airfoil on the mode shape calculations and assumed the measurements to be perpendicular to the chord.
This is because we only used the qualitative mode shape information for FEM update and thereafter used the FEM model for further .
For examples,  we just used the information that the mode at 5.8 Hz is the first symmetric bending.
A simple factor of cosine correction can be added if more fidelity in mode shapes are needed.
